CMakeLists.txt

```diff
@@ -190,55 +194,55 @@ if (APPLE)
 endif()
 
 file(GLOB BASE_SRCS
-    ${PDAL_FILTERS_DIR}/*.cpp
+    ${PDAL_FILTERS_DIR}/*.?pp
     ${PDAL_IO_DIR}/*.cpp
-    ${PDAL_KERNELS_DIR}/*.cpp
-    ${PDAL_SRC_DIR}/*.cpp
-    ${PDAL_SRC_DIR}/compression/*.cpp)
+    ${PDAL_KERNELS_DIR}/*.?pp
+    ${PDAL_SRC_DIR}/*.?pp
+    ${PDAL_SRC_DIR}/compression/*.?pp)
```

Comment:
If globbing the headers too, why skip those in the `PDAL_IO_DIR`?

Reply:
I kind of hate this in general.  This seems like something for VS to deal with, not a project like PDAL.  Maybe it wouldn't bug me if it were put off in a corner somewhere for VS where it didn't pollute the rest of the build system.

Reply:
> ฉันเกลียดสิ่งนี้โดยทั่วไป ดูเหมือนว่าบางสิ่งบางอย่างสำหรับ VS ที่จะจัดการไม่ใช่โครงการอย่าง PDAL บางทีมันอาจจะไม่ใช่ข้อผิดพลาดของฉันถ้ามันถูกนำไปวางไว้ที่มุมหนึ่งสำหรับ VS ซึ่งมันไม่ได้ทำให้ระบบการสร้างที่เหลืออยู่เสียไป